Landin page UI

Ineschvz opened the pull request.

royemosby reviewed (changes requested):
Looks pretty sharp but needs a little cleanup:

- [ ] Make sure to remove all unused components:
  - `src/exercises/week-02/directions.jsx`
  - `src/exercises/week-03/directions.jsx`
  - `src/exercises/week-04/directions.jsx`
  - `src/exercises/week-05/directions.jsx`
  - `src/exercises/week-06/directions.jsx`
  - `src/exercises/week-07/directions.jsx`
  - `src/exercises/week-08/directions.jsx`
  - `src/exercises/week-09/directions.jsx`
  - `src/exercises/week-10/directions.jsx`
  - `src/exercises/week-11/directions.jsx`
  - `src/exercises/errorBoundary.jsx`
- [ ] Some component naming issues
  - '../../exercises/week-04/studentWork': import should be StudentWork04
    - causes an `undefined` error in jsx
  - '../../exercises/week-09/studentWork': import should be StudentWork09
    - inconsistent naming compared to the rest of the weeks
- [ ] Header is missing the dropdown to navigate between weeks: should be on right-hand side.
- [ ] Make sure each week in the dropdown has a full title like week 1

- I'll put in a ticket to address the low rez hero image. It's not a high priority and I still need to make it.

Good work so far! As we move to branch-based work, it will be easier for us to discuss specific changes during PRs.

Ineschvz commented:
Pushed up changes to landin-page-ui. 

1. removed all unused components 
2. fixed studentwork naming issues 
3. added dropdown menu into header component (create a reusable component for landing page menu and header menu) 
* would love feedback** 
5. each week has a full name 

I do need to fix the dropdown icon for the header menu -> will get to it tomorrow!

royemosby reviewed (approved):
Looks great! 🚀
